cabal.project

```diff
@@ -72,5 +72,3 @@ if impl(ghc >= 9.7)
   -- this is okay
   allow-newer:
     ekg-core:text,
-    -- https://github.com/haskell-primitive/primitive-unlifted/issues/39
-    primitive-unlifted:bytestring,
```

Comment:
Seems like primitive-unlifted is not used by anything anymore, so removing it from here.

Reply:
It's a transitive dependency, but if it works then it works!